.env.rc0

```diff
@@ -0,0 +1,150 @@
+# Don't use "//" for comments in this file.
+# This file is meant to be executed as a bash script for testing.
+ENV_TYPE="production"
+
+GETH_VERBOSITY=2
+
+VM_BASED=true
+
+KUBERNETES_CLUSTER_NAME="releasecandidate0"
+KUBERNETES_CLUSTER_ZONE="us-west1-a"
+CLUSTER_DOMAIN_NAME="celo-testnet"
+
+TESTNET_PROJECT_NAME="celo-testnet-production"
+
+BLOCKSCOUT_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/blockscout"
+BLOCKSCOUT_DOCKER_IMAGE_TAG="982ffa2a1ddba3b5ccfe0915bd902ab8671ddc99"
+BLOCKSCOUT_WEB_REPLICAS=3
+# Increment this value everytime you redeploy blockscout. Or else the deployment will fail due to the
+# existing database.
+BLOCKSCOUT_DB_SUFFIX="1"
+BLOCKSCOUT_SUBNETWORK_NAME="RC0"
+
+CELOSTATS_SERVER_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celostats-server"
+CELOSTATS_SERVER_DOCKER_IMAGE_TAG="2d970c69c109ab4089bddfe50c8f75d872e6e3a3"
+CELOSTATS_FRONTEND_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celostats-frontend"
+CELOSTATS_FRONTEND_DOCKER_IMAGE_TAG="db16a1a4d9882cfed93ffa9f61cfa9f80cadfe18"
+CELOSTATS_TRUSTED_ADDRESSES=""
+CELOSTATS_BANNED_ADDRESSES=""
+CELOSTATS_RESERVED_ADDRESSES=""
+
+GETH_NODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/geth"
+# When upgrading change this to latest commit hash from the master of the geth repo
+# `geth $ git show | head -n 1`
+GETH_NODE_DOCKER_IMAGE_TAG="2d730430a7e22f5890dfb6ac74cb9f7297a3630d"
```

Comment:
Can we update this, and the one below it, to the latest commit hash from `celo-blockchain`

Reply:
Ah yes I did this but I guess I was on a branch locally, my bad